recipe/meta.yaml

```
@@ -126,9 +125,9 @@ requirements:
     - {{ pin_compatible("msys2-conda-epoch", exact=True) }}  # [win]
     # Needed by R CMD javareconf
     - make                               # [not win]
-    - {{ compiler('fortran') }}          # [not win and c_compiler!='toolchain_c']
-    - {{ compiler('cxx') }}              # [not win and c_compiler!='toolchain_c']
-    - {{ compiler('c') }}                # [not win and c_compiler!='toolchain_c']
+    - {{ fortran_compiler|default("blah") }}_{{ target_platform }} >={{ fortran_compiler_version|default("0") }}     # [not win]
```

Fortran compiler should be pinned to the version.
```suggestion
    - {{ compiler('fortran') }}     # [not win]
```

---

Why? I guess it is not backwards compatible?

---

Yes, libgfortran's SONAME changes almost every major version.